chart/values.schema.json

```
@@ -4023,6 +4023,27 @@
                     "default": null,
                     "x-docsSection": null
                 },
+                "env": {
+                    "description": "Add additional env vars to stats.",
+                    "type": "array",
+                    "default": [],
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string"
+                            },
+                            "value": {
+                                "type": "string"
+                            }
+                        },
+                        "required": [
+                            "name",
+                            "value"
+                        ],
+                        "additionalProperties": false
```

```suggestion
                        "$ref": "#/definitions/io.k8s.api.core.v1.EnvVar"
```

And we can allow "normal" env vars here because of that.